shell/platform/tizen/touch_event_handler.cc

```diff
@@ -76,36 +76,45 @@ Eina_Bool TouchEventHandler::OnTouch(void* data, int type, void* event) {
   if (type == ECORE_EVENT_MOUSE_BUTTON_DOWN) {
     self->pointer_state_ = true;
     auto* button_event = reinterpret_cast<Ecore_Event_Mouse_Button*>(event);
-    self->SendFlutterPointerEvent(kDown, button_event->x, button_event->y, 0, 0,
-                                  button_event->timestamp,
-                                  button_event->multi.device);
+    if (self->engine_->renderer()->GetWindowId() == button_event->window) {
+      self->SendFlutterPointerEvent(kDown, button_event->x, button_event->y, 0,
+                                    0, button_event->timestamp,
+                                    button_event->multi.device);
+    }
```

Comment:
I think you can return `ECORE_CALLBACK_DONE` after handling the event.
```suggestion
                                    button_event->multi.device);
      return ECORE_CALLBACK_DONE;
    }
```

Reply:
Good point! thanks! :+1: